spec/lib/evss/disability_compensation_form/data_translation_all_claim_spec.rb

```diff
@@ -1084,6 +1084,50 @@
       end
     end
 
+    context 'when there is a new disability without a classificationCode' do
+      let(:form_content) do
+        {
+          'form526' => {
+            'newPrimaryDisabilities' => [
+              {
+                'cause' => 'NEW',
+                'condition' => '  brand [new] disability { to  be } rated',
+                'primaryDescription' => 'new condition description'
+              }
+            ]
+          }
+        }
+      end
+
+      it 'should translate only the NEW disabilities' do
+        expect(subject.send(:translate_new_primary_disabilities, [])).to eq [
+          {
+            'disabilityActionType' => 'NEW',
+            'name' => 'brand new disability to be rated',
+            'serviceRelevance' => "Caused by an in-service event, injury, or exposure\nnew condition description"
+          }
+        ]
+      end
+    end
+
+    describe '#scrub_disability_condition' do
+      context 'when given a condition name' do
+        let(:condition1) { 'this is only a test' }
+        let(:condition2) { '  this    is only     a      test ' }
+        let(:condition3) { '[ this  is ] (only) a ’test’' }
+        let(:condition4) { 'this-is,only.a-test' }
+        let(:condition5) { 'this ¢is onÈly a töest' }
+
+        it 'should scrub out any illegal characters' do
+          expect(subject.send(:scrub_disability_condition, condition1)).to eq 'this is only a test'
+          expect(subject.send(:scrub_disability_condition, condition2)).to eq 'this is only a test'
+          expect(subject.send(:scrub_disability_condition, condition3)).to eq 'this is (only) a ’test’'
```

Comment:
I thought apostrophes weren't allowed? This test seems to contradict the comment in the `scrub_disability_condition` method

Reply:
Ahhhhh I see. Cool, thanks for the clarification!